### ISSUE TITLE
"Cannot find module 'tunnel'" in builds

### ISSUE THREAD

**Issue by themkat** [emacs-eask/setup-eask#22]
In Emacs dap-mode we get the following issue with setup-eask:
```
Error: Cannot find module 'tunnel'
Require stack:
- /home/runner/work/_actions/emacs-eask/setup-eask/master/dist/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.669 (/home/runner/work/_actions/emacs-eask/setup-eask/master/dist/index.js:3009:33)
    at __nccwpck_require__ (/home/runner/work/_actions/emacs-eask/setup-eask/master/dist/index.js:3[12](https://github.com/emacs-lsp/dap-mode/runs/6428449683?check_suite_focus=true#step:5:12)4:43)
    at Object.255 (/home/runner/work/_actions/emacs-eask/setup-eask/master/dist/index.js:[17](https://github.com/emacs-lsp/dap-mode/runs/6428449683?check_suite_focus=true#step:5:17)64:29)
    at __nccwpck_require__ (/home/runner/work/_actions/emacs-eask/setup-eask/master/dist/index.js:3124:43)
    at Object.41 (/home/runner/work/_actions/emacs-eask/setup-eask/master/dist/index.js:497:23)
    at __nccwpck_require__ (/home/runner/work/_actions/emacs-eask/setup-eask/master/dist/index.js:3124:43) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/emacs-eask/setup-eask/master/dist/index.js'
  ]
}
```

From what I can see, this is caused by the id of tunnel in the import statement in dist/index.js being wrong. Might be caused by a failure in:
https://github.com/emacs-eask/setup-eask/actions/runs/2317489201

To my understanding, these builds updates this file. The fix is probably just to build manually and push the file or run the pipeline again and let it handle it?

**Comment by jcs090218** [comment]
Sorry for the inconvenience! I have reverted toolchain version back to `1.7.2`.

Should fix in 414a81cc6b04b2dc242f1ad0b69c8c4032f74e76.

**Comment by themkat** [comment]
Looks like everything works again, at least when I see the newest run in dap-mode. Will close this issue now. Thank you for the quick resolution 😄
